Use this module to create your own custom cohort and to specify how cancer genomics samples are grouped and used for contrasts in iAtlas. You can also upload a file to specify the cohort and sample groups. When you are done, cohort and sample group selection will be saved, and used throughout iAtlas. **You can refer back to this module at any time to review or modify your choices.** A summary of your cohort and sample groups is shown at the bottom of this page.

You can also access the CRI iAtlas data by querying our database. We have Jupyter Notebooks detailing how to use the CRI iAtlas API to query our database:
- [Query the TCGA dataset](https://github.com/CRI-iAtlas/iatlas-notebooks/blob/main/querying_TCGA_features_and_expression.ipynb)
- Query the Immune Checkpoint Inhibition datasets [in R](https://github.com/CRI-iAtlas/iatlas-notebooks/blob/main/ici_query_iatlas_data.ipynb) and  [in Python](https://github.com/CRI-iAtlas/iatlas-notebooks/blob/main/ici_query_iatlas_data_python.ipynb)
- [Query the pseudobulk single-cell RNAseq data](https://github.com/CRI-iAtlas/iatlas-notebooks/blob/main/query_iatlas_single_cell_datasets.ipynb)
